main.tf

```diff
@@ -155,6 +155,13 @@ resource "helm_release" "websphere_liberty_operator" {
     value = var.ws_liberty_operator_namespace
   }
 
+  set {
+    name  = "installplanapprovalconfig"
+    type  = "string"
+    value = var.ws_liberty_operator_install_plan_approval
+  }
+
+  # executed to approve install-plans if var.ws_liberty_operator_install_plan_approval is Manual (if Automatic no change is expected)
```

Comment:
Any issues if script runs when install plan is set to Automatic?

Reply:
no

```
module.websphere_liberty_operator.helm_release.websphere_liberty_operator (local-exec): Install plan install-b65lh found
module.websphere_liberty_operator.helm_release.websphere_liberty_operator (local-exec): Approving install plan install-b65lh
module.websphere_liberty_operator.helm_release.websphere_liberty_operator (local-exec): installplan.operators.coreos.com/install-b65lh patched (no change)
```

An improvement could be to query only for the installPlan to be approved